doc/sphinx/proofs/writing-proofs/inductive.rst

```diff
@@ -136,26 +146,26 @@ analysis on inductive or co-inductive objects (see :ref:`inductive-definitions`)
 
    .. tacv:: destruct {+, @term}
 
-      This is a shortcut for :n:`destruct @term; ...; destruct @term`.
+      This is a shortcut for :n:`destruct @term; …; destruct @term`.
 
-   .. tacv:: destruct @term as @or_and_intropattern_loc
+   .. tacv:: destruct @term @as_or_and_ipat
 
-      This behaves as :n:`destruct @term` but uses the names
-      in :token:`or_and_intropattern_loc` to name the variables introduced in the
-      context. The :token:`or_and_intropattern_loc` must have the
-      form :n:`[p11 ... p1n | ... | pm1 ... pmn ]` with ``m`` being the
+      Behaves like :n:`destruct @term` but uses the names
```

Comment:
> it can be `destruct x as [ -> ? <- ]`

Readers can get this by following the grammar for `or_and_intropattern` though it is a couple levels down :-(.